gravitee-apim-console-webui/package.json

```diff
@@ -20,11 +20,19 @@
     "@asyncapi/web-component": "1.2.26",
```

Comment:
### :warning: Aqua detected vulnerability in your code  
**Vulnerability ID:** CVE-2024-21536   
**Check Name:** http-proxy-middleware: Denial of Service   
**Severity:** HIGH   
**Fixed Version:** 2.0.7, 3.0.3   
**Reachable Path(s) Found:** No  
**Description:** Versions of the package http-proxy-middleware before 2.0.7, from 3.0.0 and before 3.0.3 are vulnerable to Denial of Service (DoS) due to an UnhandledPromiseRejection error thrown by micromatch. An attacker could kill the Node.js process and crash the server by making requests to certain paths.  
[This comment was created by Aqua Pipeline]  
  
Read more at https://avd.aquasec.com/nvd/cve-2024-21536